lib/js/components/ProgressBar/ProgressBar.vue

```diff
@@ -0,0 +1,330 @@
+<template>
+	<div
+		:class="{
+			progressBar: true,
+			'-primary': color === PROGRESS_BAR_COLORS.PRIMARY,
+			'-warning': color === PROGRESS_BAR_COLORS.WARNING,
+			'-fail': color === PROGRESS_BAR_COLORS.FAIL,
+			'-success': color === PROGRESS_BAR_COLORS.SUCCESS,
+			'-neutral': color === PROGRESS_BAR_COLORS.NEUTRAL,
+			'-info': color === PROGRESS_BAR_COLORS.INFO,
+			'-schemeMedium': colorScheme === PROGRESS_BAR_COLOR_SCHEMES.MEDIUM,
+			'-schemeMediumNeutral': colorScheme === PROGRESS_BAR_COLOR_SCHEMES.MEDIUM_NEUTRAL,
+			'-compact': layout === PROGRESS_BAR_LAYOUTS.COMPACT,
+		}"
+	>
+		<div class="progressBar__label">
+			<div class="progressBar__labelText">{{ labelText }}</div>
+			<div v-if="labelDataExist" class="progressBar__labelDataWrapper">
+				<div v-if="labelData" class="progressBar__labelData">{{ labelData }}</div>
+				<div v-if="labelDataSupporting" class="progressBar__labelDataSupporting">
+					<span class="progressBar__labelDataSeparator">/</span>
+					{{ labelDataSupporting }}
+				</div>
+				<div v-if="labelDataSuffix" class="progressBar__labelDataSuffix">{{
+					labelDataSuffix
+				}}</div>
+			</div>
+		</div>
+		<div
+			:class="{
+				progressBar__bar: true,
+				'-small': size === PROGRESS_BAR_SIZES.SMALL,
+				'-xsmall': size === PROGRESS_BAR_SIZES.XSMALL,
+				'-noRadius': radius === PROGRESS_BAR_RADII.NONE,
+			}"
+		>
+			<div class="progressBar__results">
```

Comment:
do we need this container? Why don't we define styles on parent?

Reply:
I will remove it.